signer/sign_blinded_beacon_block.go

```diff
@@ -0,0 +1,71 @@
+package signer
+
+import (
+	"encoding/hex"
+
+	"github.com/attestantio/go-eth2-client/api"
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/bloxapp/ssv-spec/types"
+	ssz "github.com/ferranbt/fastssz"
+	"github.com/pkg/errors"
+
+	"github.com/bloxapp/eth2-key-manager/core"
+)
+
+// SignBlindedBeaconBlock signs the given beacon block
+func (signer *SimpleSigner) SignBlindedBeaconBlock(b *api.VersionedBlindedBeaconBlock, domain phase0.Domain, pubKey []byte) ([]byte, []byte, error) {
```

Comment:
we should have a unified logic for block and blinded block to avoid duplicated logic and the risk of signing a block and a blinded block becasuse of duplicated paths